style.css

```diff
@@ -332,3 +336,89 @@ header a {
   margin: 0 24px;
   padding-left: 0;
 }
+
+/* contact section */
+.contact-section {
+  display: flex;
+  width: 100%;
+  justify-content: left;
+  background: white;
+  padding: 0;
+  flex-direction: column;
+  align-items: center;
+}
+
+html,
+body {
+  max-width: 100%;
+  overflow-x: hidden;
+}
+
+.contact-section h2 {
+  font-family: var(--font-inter);
+  font-weight: 700;
+  color: #172b4d;
+  text-align: center;
+  font-size: 2em;
+  padding-top: 15vh;
+  margin-bottom: 10vh;
+}
+
+.contact-form {
```

Comment:
- [ ] Kindly adhere to the design as I mentioned above which makes your design looks perfect and good to *eyes*. Actually building the design exactly the same as the figma `UI`, actually tell any person how much you are `detail-oriented` and indeed helps you to be the more professional developer who takes care of the `clients` requirements. Please have a look at the below image and one last thing make sure that your text looks exactly the same as per the requirement, you can use `width` like **80%** or **90%** to occupy the width as per the design.

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-10 17-00-47](https://github.com/ClarenceAtim/Portfolio/assets/66476329/4ce3eab7-8459-4419-b764-bc010f654a5e)

</td>
<td>

![Screenshot from 2023-05-10 16-58-17](https://github.com/ClarenceAtim/Portfolio/assets/66476329/d1d0a04a-1973-4746-b5c6-d690edd68b9a)

</td>
</tr>
</table>


### Reminder

![image](https://github.com/geekelo/a-personal-portfolio-site/assets/66476329/c43ee2e3-782e-4080-875a-b7311226d8fa)